src/main/java/org/fundacionjala/at15/katas/langtonant/agustin/Ant.java

```diff
@@ -0,0 +1,115 @@
+package org.fundacionjala.at15.katas.langtonant.agustin;
+
+public class Ant {
+    // TODO: Make Ant Class with constructor and methods
```

Comment:
Is this comments still valid?

Reply:
It is not, I am verifing for old TODO comments in the code. It should be changed in the next commits.